bwscanner/measurement.py

```diff
@@ -76,7 +69,7 @@ def choose_file_size(self, path):
         return max(self.bw_files.keys())
 
     def choose_url(self, path):
-        return self.baseurl + '/' + self.bw_files[self.choose_file_size(path)][0]
+        return self.baseurl + self.bw_files[self.choose_file_size(path)][0]
 
```

Comment:
Is dropping the `/` intensional? Or perhaps should check that baseurl has a trailing slash in `__init__`.

Reply:
It is intentional. Yes, it seems a good solution to me to check the trailing slash in ``__init__``.
IMO, it would be even better to move URLs and files to a config file.

Reply:
As it is done in https://github.com/TheTorProject/bwscanner/pull/76/files#diff-13462b18037a179e7f8b7eafaf14bfb3, though it needs to be updated with the new urls and files